java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/ScheduledAccessTokenPublisher.java

```diff
@@ -63,7 +60,7 @@ public void onSubscribe(Flow.Subscription subscription) {
     @Override
     public void onNext(HttpResponse item) {
         if (!item.getStatus().is2xxOk()) {
-            onError(new HttpStatusException(item.getStatus(), "Error when fetching token"));
+            onError(new HttpStatusException(item.getStatus(), "Error when fetching token:"));
```

Comment:
Why this change?

Reply:
I reverted this also. I had expanded this to log more (like response http status, but this `:` was a leftover)